server/src/server.ts

```diff
@@ -907,11 +917,11 @@ export class SQFLintServer {
                                     });
                                 }
                             } else {
-                                console.log("Failed to get document", uri);
+                                this.logger.error("Failed to get document", uri);
                             }
                         } catch (e) {
-                            console.log("Failed to load " + uri);
-                            console.log(e);
+                            this.logger.error("Failed to load " + uri);
```

Comment:
might as well stuff this into a single statement?
The logger function allows for multiple args

Reply:
True